config/googlePassport.js

```diff
@@ -0,0 +1,50 @@
+const passport = require('passport');
+const User = require('../models/User');
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+
+// Environment variables
+require('dotenv').config();
+
+// Serialize user saved the providerId to the session
+passport.serializeUser((user, done) => {
+  done(null, user.providerId);
+});
+
+// Deserialize user retrieves the provider id from the session to fine user
+passport.deserializeUser(async (id, done) => {
+  try {
+    const user = await User.findOne({ providerId: id });
+    done(null, user);
+  } catch (err) {
+    console.error('ERROR - googleAuth.js - deserializeuser', err);
```

Comment:
`mext(err)`

Reply:
Next function not working due to Express not being required into file